bitbots_ball_filter/src/bitbots_ball_filter/ball_filter.py

```diff
@@ -16,22 +17,28 @@ def __init__(self):
         creates Kalmanfilter and subscribes to messages which are needed
         """
         rospy.init_node('ball_filter')
-        #creates kalmanfilter with 4 dimensions
+
+        # creates kalmanfilter with 4 dimensions
         self.kf = KalmanFilter(dim_x=4, dim_z=2, dim_u=0)
         self.filter_init = False
```

Comment:
By the way (and not really related to this PR), I am always confused by the name `filter_init` because to me, it sounds as if a value of `True` means that the filter should be initialized in the next step or something. Hence I would prefer to rename it to `fiter_initialized`. But that's just my two cents.